HW#1 – Vasilii Shabanov

shabspb opened the pull request:
Decision homework #1 - Cubes.
In this solution i used a preprocessor SCSS, also BEM, bootstrap-reboot and gulp.

nvkuznetsova commented on Weather/debug.log:
```diff
@@ -0,0 +1,114 @@
+[0918/191341.899:ERROR:crash_report_database_win.cc(469)] failed to stat report
```
It's better to add all log files and folders to `.gitignore` file

nvkuznetsova commented on Weather/dist/debug.log:
```diff
@@ -0,0 +1,6 @@
+[0919/134146.273:ERROR:crash_report_database_win.cc(469)] failed to stat report
```
There is no need to add the `dist` folder to the repo. Please, add `dist` folder to `.gitignore`

nvkuznetsova commented on Weather/src/sass/blocks/_widget.scss:
```diff
@@ -0,0 +1,92 @@
+
+        &__wrapper {
+            display: flex;
+            justify-content: space-evenly;
```
Value `space-evenly` is not supported in ie11. You can check CSS properties and their values ​​for cross-browser compatibility here https://caniuse.com/. You can use `space-around` or `space-between` instead.

nvkuznetsova commented:
The widget looks nice in the latest browsers. Good job, keep going! 👍
You can also try to use semantic tags to improve HTML. No need to do this in the homework. Just a little advice for training 😊 
Also, please, pay more attention to `.gitignore` file. Don't forget to add log and build files to  `.gitignore`.  Usually these files are not related to the changes you make in code. And without them, it is much easier to do a codereview.  
To make it clear how to run your code, you can add the readme.md file the Weather folder and describe here the steps how to run your app

shabspb commented:
Thanks for your feedback, it is very important to me.
I have modified .gitignore file and added all files with the .log extension in that file. I also added the dist folder there. And moved README.md file to weather folder with new description. Also i made necessary changes to CSS to support my widget by IE browser.
But I don't understand what steps I need to describe in README.md file to launch the app? Please check my version and tell me if I understood you correctly =)  I understand that I wrote the description incorrectly, but I do not understand how to do it differently.

nvkuznetsova commented:
You could just create a separate readme.md file in Weather folder not to affect the existing file) 
Describe steps to launch your app means to list commands and scripts so that anyone can understand from these steps how to launch your app. For example, to run your app correctly it is necessary to run `npm install` first. You can describe what scripts to run to compile sass to CSS, to run your app with GULP.  
Here the example of readme from my pet-project
![image](https://user-images.githubusercontent.com/31770506/94467269-5ae6dc00-01cb-11eb-911d-483b12cc89c9.png)
No need to fix this, this is just for your information 😊
